src/cephadm/cephadm

```diff
@@ -759,6 +759,24 @@ def infer_fsid(func):
 
     return _infer_fsid
 
+def get_last_local_ceph_image():
+    """
+    :return: The most recent local ceph image (already pulled)
+    """
+    out, _, _ = call_throws(
+        [container_path, 'images', '--format', '{{ .ID }} {{.Repository}} {{.Tag}}'])
+    out_lines = out.splitlines()
+    for out_line in out_lines:
+        id, repository, tag = out_line.split()
+        out, _, _ = call_throws([container_path, 'inspect', str(id)])
```

Comment:
If we want to avoid calling `inspect` here we have to reopen https://github.com/ceph/ceph-container/pull/1604 (or equivalent) because `podman --filter` doesn't support wildcards or filter by key only (only docker supports that)

Reply:
reopened.. it would be nice to avoid all of the inspect calls

Reply:
Pushed a new commit that avoids the `inspect` call by using the `ceph=True` label.